Makefile

```diff
@@ -44,8 +44,8 @@ ${VENV_ACTIVATE}: requirements.txt
 
 install-dependencies: ${VENV_ACTIVATE}
 
-docker/generated.mk: docker/generate_makefile.py $(wildcard fuzzers/*/variants.yaml) ${VENV_ACTIVATE}
-	source ${VENV_ACTIVATE} && python3 $< > $@
+docker/generated.mk: experiment/build/generate_makefile.py $(wildcard fuzzers/*/variants.yaml) ${VENV_ACTIVATE}
```

Comment:
We're moving this to experiment/build now? Why?

Reply:
The related `generate_makefile.py`, `generate_cloudbuild.py` and `docker_images.py` are in the same directory. Should `generate_makefile` be moved back to `docker/`?

Reply:
If you look at .github/workflows/fuzzers.yml it's configured to only run if certain directories are changed, including docker/. So yes I'd like it to move back, unless that config and anything else in src_analysis is updated to deal with new locations.

Reply:
Moved back.